src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -343,12 +342,18 @@ private void AddNameAndTypeArgumentsOrParameters(INamedTypeSymbol symbol)
                 symbolName = symbol.Name;
             }
 
+            var isAnoymousDelegate = symbol.TypeKind == TypeKind.Delegate && !symbol.CanBeReferencedByName;
             if (format.MiscellaneousOptions.IncludesOption(SymbolDisplayMiscellaneousOptions.UseErrorTypeSymbolName) &&
                 partKind == SymbolDisplayPartKind.ErrorTypeName &&
                 string.IsNullOrEmpty(symbolName))
             {
                 builder.Add(CreatePart(partKind, symbol, "?"));
             }
+            else if (isAnoymousDelegate)
+            {
+                var name = $"<anonymous delegate>";
```

Comment:
>$ [](http://example.com/codeflow?start=27&length=1)

Minor: `$` is not needed.

And perhaps just inline the value in the call below.

Reply:
sure.